### PR TITLE
feat(athena): chunking sqs

### PR DESCRIPTION
**Pull request by thomasyopes**
Ref: #2330

### Description

- splitting sqs into chunks and sleeping

### Testing

- Staging
  - [x] test 10k patient file
- Production
  - [ ] test 10k patient file

### Release Plan

- [ ] Merge this

### CODE REVIEW

File: packages/core/src/command/patient-import/patient-import-cloud.ts [hunk 1]
```diff
@@ -123,44 +129,50 @@ export class PatientImportHandlerCloud implements PatientImportHandler {
...
+                JSON.stringify(processPatientCreateRequest),
+                {
+                  fifo: true,
+                  messageDeduplicationId: uuidv7(),
+                  messageGroupId: cxId,
+                }
+              );
+            } catch (error) {
+              const msg = `Failure while sending payload to patient create queue @ PatientImport`;
+              log(`${msg}. Cause: ${errorToString(error)}`);
```
Comment (Goncharo) [inline packages/core/src/command/patient-import/patient-import-cloud.ts:158]: hypernit: why create a const for a msg that's used once?

**Review by Goncharo — APPROVED** [review]
LGTM - couple nits

File: packages/core/src/command/patient-import/patient-import-cloud.ts (hunk 1 above)
Comment (thomasyopes) [reply packages/core/src/command/patient-import/patient-import-cloud.ts:158]: you mean vs. one-line or vs. a "var" or something?

File: packages/core/src/command/patient-import/patient-import-cloud.ts [hunk 2]
```diff
@@ -26,6 +28,10 @@ const region = Config.getAWSRegion();
 const lambdaClient = makeLambdaClient(region);
 const sqsClient = new SQSClient({ region });
 
+// 20 / second -> 18000 in 15min
```
[the remark this answers is not in the document]
Comment (thomasyopes) [reply packages/core/src/command/patient-import/patient-import-cloud.ts]: Done.

File: packages/core/src/command/patient-import/patient-import-cloud.ts [hunk 3]
```diff
@@ -26,6 +27,10 @@ const region = Config.getAWSRegion();
 const lambdaClient = makeLambdaClient(region);
 const sqsClient = new SQSClient({ region });
 
+// CURRENT: 5 chunks every 250ms is 20 patients per second -> 18000 in 15min (the lambda timeout)
+const sleepBetweenPatientCreateChunksInMillis = 250;
```
Comment (thomasyopes) [inline packages/core/src/command/patient-import/patient-import-cloud.ts:31]: for later: dayjs

**Pull request merged**